packages/davinci-client/src/lib/config.slice.ts

```diff
@@ -40,22 +40,15 @@ export const configSlice = createSlice({
     set(state, action: PayloadAction<DaVinciConfig>) {
       state.clientId = action.payload.clientId || '';
       state.redirectUri = action.payload.redirectUri || `${location.origin}/handle-redirect`;
-      state.responseType = action.payload.responseType || 'code';
-      state.scope = action.payload.scope || 'openid';
-
-      if (!action.payload.serverConfig?.baseUrl) {
-        state.serverConfig = {
-          baseUrl: '',
-        };
-      } else if (action.payload.serverConfig?.baseUrl.endsWith('/')) {
-        state.serverConfig = {
-          baseUrl: action.payload.serverConfig?.baseUrl,
-        };
+      if ('responseType' in action.payload) {
+        state.responseType = action.payload.responseType;
       } else {
-        state.serverConfig = {
-          baseUrl: action.payload.serverConfig?.baseUrl.concat('/'),
-        };
+        state.responseType = 'code';
       }
+      state.scope = action.payload.scope || 'openid';
+    },
+    setEndpoints(state, action: PayloadAction<Endpoints>) {
```

Comment:
I'm not entirely sold on having this be a different reducer. We should be able to just include the endpoints in the regular `set` reducer, since it's all going in the same place and we already have a president with the JS SDK: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/packages/javascript-sdk/src/config/interfaces.ts#L91 (although named `paths` and not `endpoints`).

Unless, I'm missing something that requires this be a separate reducer.

Reply:
I can make it happen in `set` thats fine. I've left it as endpoints unless you feel strongly about `paths`. I've personally never loved "paths" naming but I'm happy to discuss something that may be better than `endpoints`.

Reply:
That's fine. We can use `endpoints` here; I agree about `paths`.